### ISSUE TITLE
Dashboard shows 0 proxy hosts for new admin user, while lists shows all as expected

### ISSUE THREAD

**Issue by AcidGreg** [NginxProxyManager/nginx-proxy-manager#2014]
**Checklist**
- Have you pulled and found the error with `jc21/nginx-proxy-manager:latest` docker image?
  - Yes
- Are you sure you're not using someone else's docker image?
  - Yes
- Have you searched for similar issues (both open and closed)?
  - Yes: #1679 exists but was closed probably due to inability to reproduce

**Describe the bug**
After adding a new admin user account, and logging in with it, the proxy host count on the dashboard is 0, while there are indeed defined proxy hosts. Clicking the icon gets us to the correct list, however, with all hosts shown. Just the dashboard view count is wrong. This has happened on 2/2 deployments within the last week. It might be important to note the following to reproduce: In both cases, the new admin user was added and then immediately a password was set by the original admin user, then logout, login with the new one and the issue appears.

**Nginx Proxy Manager Version**
v2.9.18

**To Reproduce**
When some proxy hosts are defined and you are logged in with the original admin account and no other users yet exist:
1. Add a new admin account
2. Set a password for the new admin account
3. Log out
4. Log in with the new admin account
5. The problem should be visible: The Proxy hosts count is 0 when some other number is expected.
6. Click on Proxy Hosts
7. You should now see what you expect to see in this view - the correct list of defined proxy hosts.

**Expected behavior**
The expected behavior would be to see the correct count of defined proxy hosts in the dashboard view and not 0.

**Screenshots**

![image](https://user-images.githubusercontent.com/4808842/164041014-5b493dfa-5f3e-4dbf-8bd6-3915d9b566a4.png)
![image](https://user-images.githubusercontent.com/4808842/164041139-fb915bae-0ebf-4fd0-afdf-4df926fbe687.png)

Here are the two defined admin users: Norbert is the original admin defined after installation.
![image](https://user-images.githubusercontent.com/4808842/164041071-8ecd1050-7598-4232-a3d4-89460daa7f76.png)

**Operating System**
Docker image on Ubuntu (18.04 and 20.04) x86-64

**Additional context**
Using latest Chrome browser. Did not feel the need to try with others.

**Comment by robonuka** [comment]
I see the same problem!

**Comment by markpsullivan** [comment]
Same issue here.

**Comment by dannykorpan** [comment]
Same issue here

**Comment by jc21** [comment]
Yeah this is an age old problem with the UI caching the result of a different API result. I'm not working on this UI in favour of the react version on the `v3` branch.

**Comment by teslazap** [comment]
my counter only counts proxy hosts I created with my user (5), even though I can see and manage all of them (8)
the other user's counter shows 3, but again he can see and manage all of them (8)

is this the intended behavior? it's a bit confusing in my opinion, shouldn't the counter match what I see?

**Comment by axheli** [comment]
Same here. Any news on this ?

**Comment by robonuka** [comment]
I think the solution is to forget about this soft.
Use alternative products instead. It's without WEB-UI, but simple, stable and usefull.

**Comment by Kalmah85** [comment]
Same here.

**Comment by partybence** [comment]
Still present in v2.12.6 - noticed it with redirection-hosts
I'm suspecting it may be that in backend/internal/redirection-host.js, in the getCount function there's a statement:
```
if (visibility !== 'all') {
	 query.andWhere('owner_user_id', user_id);
}
```
which, it seems, doesn't see the visibility permission as it should.
Another enpoint, /api/users/me?expand=permissions does show it though, so the visibility permission is set to 'all' - the hosts themselves are visible to the user, only the dashboard shows an improper count - so it seems, the two endpoints take this information from different sources. Unfortunately I haven't managed to follow the line further than /lib/express/jwt-decode.js, if I'm not mistaken, this is where the access parameters should originate.
